src/frida/repl.py

```diff
@@ -51,6 +56,13 @@ def _start(self):
                 self._update_status("Failed to load script: {error}".format(error=e))
                 self._exit(1)
                 return
+
+            if self._user_script is not None and self._watch:
+                monitor = frida.FileMonitor(self._user_script)
+                monitor.on('change', self._on_change)
+                monitor.enable()
+                self._script_monitor = monitor
+
```

Comment:
I'm not sure what you are suggesting. At the moment I've assumed that you only want --watch to work if you specified it and -l at the command line.